source/clear-linux/concepts/swupd-about.rst

```diff
@@ -1,80 +1,119 @@
+of new enhancements and security fixes. Refer to :ref:`security` documentation
+for more information.
 
-While this may seem like a limitation, the method has several benefits:
+Learn how to update your system using :ref:`swupd <swupd-guide>`.
 
-* In a cloud environment, |CL| allows you to run the exact same software
-  for all container hosts in a cluster.
 
-* |CL| intends to help developers create custom |CL| derivatives. Using
-  :ref:`mixer <mixer>`, system administrators can focus on customizing their deployments while staying on a controlled update stream.
+Update efficiency 
+-----------------
 
-* Regular updates ensure :ref:`security` is tighter, so it is
-  much easier to monitor and update patches.
+Because :command:`swupd` operates at the individual file-level instead of a
+package-level, |CL| updates are able to be small and fast. 
 
-Learn how to update your system using :ref:`swupd <swupd-guide>`.
+|CL| updates are generated using the :ref:`mixer <mixer-about>` tool. Mixer
+calculates the difference between two |CL| versions and makes available
+*binary deltas*, which contain only the changed portion of files. This *binary
+delta technology* [1]_ means :command:`swupd` on |CL| systems only needs to
+download and apply a small fraction of a package in order to receive an
+update.
+
+
+While similar update technologies exist, they are seldomly used in practice.
+In other Linux\* distributions, updates to a particular software package
+typically results in the whole software package being downloaded again and
+replaced on the system - even when the net change is just one line of code. 
+
+The :ref:`mixer <mixer-about>` tool additionally computes updates files in
+multiple compression formats, allowing :command:`swupd` to utilize the most
+efficiently compressed format for a |CL| system in order to minimize the cost
+to update.
+
+
+Update integrity 
+----------------
+
+:command:`swupd` operates against a published manifest of files for a
+particular |CL| version that contains the unique hash of each file. This is
+the basis of the :command:`swupd verify` subcommand which allows a |CL| system
+to check for, and remediate, any discrepancies to system files. The
+:command:`swupd verify` can be especially useful for software developers to
+return to a known filesystem state.
+
+
+
+Bundles
+=======
+
+|CL-ATTR| approaches software management differently than other distributions
+of Linux-based operating systems. 
+
+Instead of deploying granular software packages, |CL| uses the concept of
+bundles with pre-associated software. Each bundle encapsulates a particular
+use-case, which is enabled by composing all the required upstream open-source
+projects and packages into one logical unit. 
+
+This bundle-based approach offers some unique advantages.
+
+- Installing software for a particular functionality will come "bundled" with 
+  all of its runtime dependencies, avoiding a situation where you have to hunt
```

Comment:
Simplify: 
- Software for a particular functionality comes bundled with all runtime dependencies [omit 'avoiding....']

Reply:
Revised.